components/carbon-authenticators/saml2-sso-authenticator/org.wso2.carbon.identity.authenticator.saml2.sso.ui/pom.xml

```diff
@@ -115,7 +115,9 @@
                             org.wso2.carbon.identity.authenticator.saml2.sso.stub;
                             version="${carbon.identity.package.import.version.range}",
                             org.wso2.carbon.identity.authenticator.saml2.sso.stub.types;
-                            version="${carbon.identity.package.import.version.range}"
+                            version="${carbon.identity.package.import.version.range}",
+                            org.wso2.carbon.identity.application.authentication.framework.model.xsd;
```

Comment:
org.wso2.carbon.identity.application.authentication.framework.model.xsd does not have any version and org.wso2.carbon.identity.application.authentication.framework.model.* does not work